src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -48,16 +51,20 @@
     private final TopicRepository topicRepository;
     private final PartitionResolver partitionResolver;
     private final Enrichment enrichment;
+    private final FeatureToggleService featureToggleService;
 
     @Autowired
     public EventTypeController(final EventTypeRepository eventTypeRepository,
                                final TopicRepository topicRepository,
                                final PartitionResolver partitionResolver,
-                               final Enrichment enrichment) {
+                               final Enrichment enrichment,
+                               final FeatureToggleService featureToggleService)
+    {
```

Comment:
Another good point for coding convention discussion :-D
Starting braces from a new line :scream: